### PR TITLE
Create doc for SSO Dashboard Extension

### PR DESCRIPTION
**Pull request by angelaLazar**
https://trello.com/c/1Epql5Ed/533-documentation-for-the-auth0-sso-dashboard-extension

### CODE REVIEW

File: articles/extensions/sso-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+![Dashboard Settings](/media/articles/extensions/sso-dashboard/settings.png)
+
+You will then need to enter the following fields for the new application:
+
+**Type**: This field is a dropdown where you select the either SAML, OpenID-Connect, or WS-Federation depending on the type of application.
+**Application**: This is the application name of the application you have created that you wish to associate the login of users.
+**Name**: The name of the new application you are adding.
+**Logo**: Enter the url of the logo you wish to user as an icon for the application.
+**Callback**: This is one of the **Allowed Callback URLs** under your [Client Settings](${manage_url}/#/clients) of the application.
+**Connection**: Select the connection type from the dropdown. You can add/edit your available connection types in the [Connections section of the Auth0 Management dashboard](${manage_url}/#/co.nnections/database).
```
Comment (sandrinodm) [inline articles/extensions/sso-dashboard.md]: Make it clear that this is optional. If a connection is not set and the user is not logged in yet, they will see the Auth0 Login page (with the lock)

File: articles/extensions/sso-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+description: This page explains how to configure and utilize the SSO Dashboard Extension
+---
+
+# Auth0 Extension: Single Sign-On (SSO) Dashboard
+
+The **SSO Dashboard** extension allows you to create a dashboard with multiple enterprise applications that can be enabled for single sign-on for your users for login.
+
+[View this Extension on GitHub](https://github.com/auth0-extensions/auth0-sso-dashboard-extension)
+
```
Comment (sandrinodm) [inline articles/extensions/sso-dashboard.md:14]: Before jumping into configuration we need to do 2 things:

 - Explain that 2 types of users can login. Dashboard Admins can login to configure the applications and end users can login to see the dashboard and login to the applications.
 - Explain that in order to achieve this we need to:
    - Create a client (https://auth0.com/docs/extensions/delegated-admin#create-a-client)
    - Enable connections for the client: https://auth0.com/docs/extensions/delegated-admin#enable-connections-on-the-client. This section should not be copy pasted from delegated admin. We just need to explain that every connection enabled for the client will allow users to login to the SSO dashboard
 - Explain that there will be 2 clients: https://auth0.com/docs/extensions/delegated-admin#install-the-extension
 - Explain that there are 2 urls
   - https://YOU.REGION.webtask.io/auth0-sso-dashboard/admins/login => where admins can login
   - https://YOU.REGION.webtask.io/auth0-sso-dashboard/admins/login => where end users can login

**Review by sandrinodimattia — CHANGES_REQUESTED** [review]

File: articles/extensions/sso-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+| --- | --- |
+| USA | `https://${account.tenant}.us.webtask.io/auth0-sso-dashboard/admins/login` |
+| Europe | `https://${account.tenant}.eu.webtask.io/auth0-sso-dashboard/admins/login` |
+| Australia | `https://${account.tenant}.au.webtask.io/auth0-sso-dashboard/admins/login` |
+
+The login URL for **Users**:
+
+| Location | Allowed Callback URL |
+| --- | --- |
+| USA | `https://${account.tenant}.us.webtask.io/auth0-sso-dashboard/users/login` |
```
Comment (sandrinodm) [inline articles/extensions/sso-dashboard.md]: Remove `/users` from all urls. The url for users is just `/login`

File: articles/extensions/sso-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+To login into the dashboard:
+
+For **Admins**:
+
+`https://${account.tenant}.<REGION>.webtask.io/auth0-sso-dashboard/admins/login` or through the Auth0 Management Dashboard.
+
+For **Users**:
+
+`https://${account.tenant}.<REGION>.webtask.io/auth0-sso-dashboard/users/login`
```
Comment (sandrinodm) [inline articles/extensions/sso-dashboard.md]: Remove `/users`

File: articles/extensions/sso-dashboard.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+![Dashboard Settings](/media/articles/extensions/sso-dashboard/settings.png)
+
+You will then need to enter the following fields for the new application:
+
+**Type**: This field is a dropdown where you select the either SAML, OpenID-Connect, or WS-Federation depending on the type of application.
+**Application**: This is the application name of the application you have created that you wish to associate the login of users.
+**Name**: The name of the new application you are adding.
+**Logo**: Enter the url of the logo you wish to user as an icon for the application.
+**Callback**: This is one of the **Allowed Callback URLs** under your [Client Settings](${manage_url}/#/clients) of the application.
+**Connection** *Optional*: Select the connection type from the dropdown. You can add/edit your available connection types in the [Connections section of the Auth0 Management dashboard](${manage_url}/#/co.nnections/database).  If a connection is not set and the user is not logged, the user will see the Auth0 Login page.
```
Comment (sandrinodm) [inline articles/extensions/sso-dashboard.md]: The url here seems broken. There is a dot in connections

**Pull request merged**